DM-1013 Fix properties in OPC UA failure handling (#869)
SoftwareAG/c8y-docs: content/protocol-integration/opcua-bundle/device-protocols.md

If a custom action fails, a retry mechanism will be processed. This is configured in the application YAML file, and the queues will be stored in the event repository.

Queues are collections of failed custom actions, including the complete HTTP request of this custom action. Each entry of the queue is one failed custom action. 

A background scheduler task will retry each queue up to the number of _maxRetries_. If _maxRetries_ is reached the queue will be stored as a permanently failed queue in the event repository.

> All elements of the queue will be retried after the retry delay has passed. In effect, the count of the elements in the queue will be decreasing with each successful retried custom action. 

Reviewer: ```suggestion
All elements of the queue are retried after the retry delay has passed. In effect, the count of the elements in the queue is decreases with each successful retried custom action. 
```

> In the gateway configuration (see also [Gateway Configuration: Additional customization](#additional-customizations)) the following parameters can be configured:

Reviewer 2: property group information is not mentioned.
gateway.mappingExecution.http.failureHandling

|<div style="width:200px">Setting</div>|Value format|Example |Description|																				
|-----------|------------|--------|-----------|

> |enabled  | boolean (true/false) | true | Activate or deactivate the fail over for custom actions |

Reviewer 2: Information about the defaults are missing.

> |maxRetries| number | 5 | Number of retries for failed queues; if the maximum is reached the queue will be saved as permanently failed and never retried again, default is 5.|

Reviewer: ```suggestion
|maxRetries| number | 5 | Number of retries for failed queues. If the maximum is reached the queue is saved as permanently failed and never retried again. Default is 5.|
```

> |noRetryHttpCodes| Comma-separated list of HTTP response codes | 400,500 |  If retries are enabled (failureHandling.enabled=true), this setting allows retries to be skipped for certain HTTP response codes. In the given example invocations to the HTTP endpoint being answered with `400 Bad Request` or `500 Internal server error` would not be reattempted.|

Reviewer: ```suggestion
|noRetryHttpCodes| Comma-separated list of HTTP response codes | 400,500 |  If retries are enabled (failureHandling.enabled=true), this setting allows retries to be skipped for certain HTTP response codes. In the given example invocations to the HTTP endpoint that receive a `400 Bad Request` or `500 Internal server error` response are skipped.|
```
